ddtrace/llmobs/_llmobs.py

```diff
@@ -108,22 +108,22 @@ def _llmobs_tags(span: Span, meta: Dict[str, Any]) -> List[str]:
             tags.append("error_type:%s" % err_type)
         return tags
 
-    def submit_llmobs_span(self, span):
+    def submit_llmobs_span(self, span: Span) -> None:
         """Generate and submit an LLMObs span event to be sent to LLMObs."""
-        meta = json.loads(span._meta.pop("ml_obs.meta", {}))
+        meta = json.loads(span._meta.pop("ml_obs.meta", "{}"))
         if span.error:
             meta["error.message"] = span.get_tag("error.message")
-        metrics = json.loads(span._meta.pop("ml_obs.metrics", {}))
+        metrics = json.loads(span._meta.pop("ml_obs.metrics", "{}"))
         span_event = self._llmobs_span_event(span, meta, metrics)
         log.debug("Submitting span to LLMObs: %s", span)
         self._writer.enqueue(span_event)
 
-    def _llmobs_span_event(self, span, meta, metrics):
+    def _llmobs_span_event(self, span: Span, meta: Dict[str, Any], metrics: Dict[str, Any]) -> Dict[str, Any]:
         """Span event object structure."""
         return {
             "trace_id": "{:x}".format(span.trace_id),
             "span_id": str(span.span_id),
-            "parent_id": str(span.parent_id or ""),
+            "parent_id": str(self._get_llmobs_parent_id(span) or ""),
```

Comment:
If `self._get_llmobs_parent_id(span)` returns `None`, i think `parent_id` will be set to `str(None)` instead of `""`. 

Is it intended for it to be `""` when there's no parent?

Reply:
Good question! Using the `or` here returns `""` if `self._get_llmobs_parent_id(span)` returns `None`. If there's no parent, then this will always evaluate to `""`.